components/list/list-item.js

```diff
@@ -16,6 +18,7 @@ class ListItem extends RtlMixin(LitElement) {
 	static get properties() {
 		return {
 			breakpoints: { type: Array },
+			checked: { type: Boolean, reflect: true },
```

Comment:
Not sure whether this should be `checked` or `selected`.  The `checked` term is specific to checkboxes, and if there were a such thing as single selection, it wouldn't be quite right I think.